src/main/java/main/EmployeesProcessor.java

```diff
@@ -0,0 +1,388 @@
+                    addressOfEmployees.add(employeeIterator.next().getAddress());
+                }
+                break;
+            }
+            case STREAM:{
+               employeeList
+                       .forEach(System.out::println);
+                addressOfEmployees = employeeList.stream()
+                        .map(Employee::getAddress)
+                        .collect(Collectors.toSet());
+                break;
+            }
+        }
+        return addressOfEmployees;
+    }
+
+    /**
+     * TASK #9
+     * Витягнути колекцію імен працівників так, щоб нова колекція включала в себе 5 імен
+     */
+    public List<String> getFiveEmployeesName(List<Employee> employeeList, ExecutionType executionType){
+        List<String> fiveEmployeesName = new ArrayList<>();
+        switch (executionType){
+            case FOREACH:{
+                for (Employee e:employeeList){
+                    fiveEmployeesName.add(e.getFirstName());
+                    if (fiveEmployeesName.size() == 5)
+                        break;
+                }
+                break;
+            }
+            case ITERATOR:{
+                Iterator<Employee> employeeIterator = employeeList.iterator();
+                while (employeeIterator.hasNext()){
+                    fiveEmployeesName.add(employeeIterator.next().getFirstName());
+                    if (fiveEmployeesName.size() == 5)
+                        break;
+                }
+                break;
+            }
+            case STREAM:{
+                fiveEmployeesName = employeeList.stream()
+                        .map(Employee::getFirstName)
+                        .limit(5)
+                        .collect(Collectors.toList());
+            }
+            break;
+        }
+        return fiveEmployeesName;
+    }
+
+    /**
+     * TASK #10
+     * Порахувати суму віку всіх працівників
+     */
+    public Integer getSumOfAge(List<Employee> employeeList, ExecutionType executionType){
+        Integer sumOfAge = 0;
+        switch (executionType){
+            case FOREACH:{
+                for (Employee e:employeeList){
+                    sumOfAge += e.getAge();
+                }
+                break;
+            }
+            case ITERATOR:{
+                Iterator<Employee> employeeIterator = employeeList.iterator();
+                while (employeeIterator.hasNext()){
+                    sumOfAge += employeeIterator.next().getAge();
+                }
+                break;
+            }
+            case STREAM:{
+                sumOfAge = employeeList.stream()
+                        .map(Employee::getAge)
+                        .reduce(0,Integer::sum);
+            }
+            break;
+        }
+        return sumOfAge;
+    }
+
+    /**
+     * TASK #11
+     * Перевірити, чи всі працівники старше 18ти
+     */
+    public boolean isAllEmployeesOldestEighteen(List<Employee> employeeList, ExecutionType executionType){
```

Comment:
not "Oldest" but "Older") Just an English mistake - you can change it or not :) As you wish